src/Microsoft.ML.Transforms/Text/WordEmbeddingsExtractor.cs

```diff
@@ -628,7 +628,7 @@ private string EnsureModelFile(IHostEnvironment env, out int linesToSkip, WordEm
                 {
                     string dir = kind == WordEmbeddingEstimator.PretrainedModelKind.SentimentSpecificWordEmbedding ? Path.Combine("Text", "Sswe") : "WordVectors";
                     var url = $"{dir}/{modelFileName}";
-                    var ensureModel = ResourceManagerUtils.Instance.EnsureResourceAsync(Host, ch, url, modelFileName, dir, Timeout);
```

Comment:
Sorry I missed this in the earlier reviews. Why is it necessary to change Host to env?

Reply:
No worries! In all the other cases where `EnsureResourceAsync` is used, the provided `IHostEnvironment env` is used, not `IHost Host`. This is so that the function can get the proper environment. This particular usage of `EnsureResourceAsync` was the only case where `Host` was being provided, so I fixed it.